src/pages/who-we-are/_DerivNumbers.tsx

```diff
@@ -125,7 +125,15 @@ const DerivNumbers = () => {
                 >
                     {deriv_numbers.map((number) =>
                         number.map(({ count, title }) => (
-                            <Flex key={count} fd="column" height="unset">
+                            <Flex
+                                key={
+                                    typeof count === 'string'
+                                        ? count
+                                        : count.props.translate_text
```

Comment:
```suggestion
                                    typeof count === 'string'
                                        ? count
                                        : count.props.translate_text
```
@eduard-deriv, would this `count?.props?.translate_text || count` work?

Reply:
Hi @jim-deriv , you are right, I have redone it as you said, but on [deriv.com](http://deriv.com/) we use buildTime Api and don’t need to use an optional.